cmd/server/cmd/server.go

```diff
@@ -255,6 +263,31 @@ func runServer(sa *serverArgs, printf, fatalf shared.FormatFn) {
 	printf("Starting Config API server on port %v", sa.configAPIPort)
 	go configAPIServer.Run()
 
+	var monitoringListener net.Listener
+	// get the network stuff setup
+	if monitoringListener, err = net.Listen("tcp", fmt.Sprintf(":%d", sa.monitoringPort)); err != nil {
+		fatalf("Unable to listen on socket: %v", err)
+	}
+
+	// NOTE: this is a temporary solution for provide bare-bones debug functionality
+	// for mixer. a full design / implementation of self-monitoring and reporting
+	// is coming. that design will include proper coverage of statusz/healthz type
+	// functionality, in addition to how mixer reports its own metrics.
+	srvMux := http.NewServeMux()
+	srvMux.Handle(sa.metricsPath, promhttp.Handler())
```

Comment:
It uses the default gatherer, which includes two basic collectors for free:

> NewProcessCollector returns a collector which exports the current state of process metrics including cpu, memory and file descriptor usage as well as the process start time for the given process id under the given namespace.

> NewGoCollector returns a collector which exports metrics about the current go process.

I think this is good enough to start (especially for a bridge solution to a more full-fledged solution).